controllers/flp/flp_common_objects.go

```diff
@@ -282,7 +284,29 @@ func (b *builder) ConfigMap() (*corev1.ConfigMap, string, error) {
 	return &configMap, digest, nil
 }
 
-func (b *builder) GetJSONConfig() (string, error) {
+func (b *builder) DynamicConfigMap() (*corev1.ConfigMap, string, error) {
+	configStr, err := b.GetDynamicJSONConfig()
+	if err != nil {
+		return nil, "", err
+	}
+
+	configMap := corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      b.dynamicConfigMapName(),
+			Namespace: b.info.Namespace,
+			Labels:    b.labels,
+		},
+		Data: map[string]string{
+			configFile: configStr,
+		},
+	}
+	hasher := fnv.New64a()
+	_, _ = hasher.Write([]byte(configStr))
+	digest := strconv.FormatUint(hasher.Sum64(), 36)
+	return &configMap, digest, nil
+}
```

Comment:
if I'm correct, the digest here isn't used, as digests are used to restart pods, and in the case of dynamic config we precisely don't want to restart pods?
If it's not used anywhere I'd rather not generate it, also to not bring confusion about how the reconciliation works

Reply:
Fixed, thanks.